backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -2,7 +2,7 @@
 from fastapi import Response
 from datetime import datetime, timedelta, date
 from fastapi import APIRouter, HTTPException
-from ...commons.ocp import getData
+from ...commons.ocp import getData, getFilterData
```

Comment:
Some modules use this relative `...commons` syntax while others use the rooted `app.api.v1.commons.constants`. I think I prefer the rooted form since it's easier to see exactly what's being referenced ... but, at least in your changes, it'd be nice to be consistent. (And you've added both in this PR.)

Reply:
Updated in all the files